julia.html.markdown

```diff
@@ -669,14 +669,14 @@ Lion    <: Cat  # => true
 Panther <: Cat  # => true
 
 # Defining a function that takes Cats
-function pet_cat(cat::Cat)
+function petCat(cat::Cat)
     println("The cat says $(meow(cat))")
 end
-# => pet_cat (generic function with 1 method)
+# => petCat (generic function with 1 method)
 
-pet_cat(Lion("42")) # => The cat says 42
+petCat(Lion("42")) # => The cat says 42
```

Comment:
Function name needs to be changed. Refer https://github.com/adambard/learnxinyminutes-docs/pull/3256#issuecomment-427193194